controller/src/main/java/org/jboss/as/controller/ResourceBuilder.java

```diff
@@ -82,6 +83,8 @@
 
     ResourceBuilder addCapabilities(Capability... capability);
 
+    ResourceBuilder addAdditionalPackages(RuntimePackageDependency... packages);
```

Comment:
Thanks for javadoc, will add it. 
Yes, it could be renamed to addAdditionalRuntimePackageDependencies. They are added in addition to the required one defined in module.xml. addAdditional could be rename to: addExtra Seems less redundant than addAdd... and shorter.

Reply:
Extra is too vague actually, I will keep addAdditionalRuntimePackageDependencies